reframe/core/schedulers/slurm.py

```diff
@@ -91,8 +91,35 @@ def __init__(self):
         self._update_state_count = 0
         self._completion_time = None
 
-    @property
-    def completion_time(self):
+    def completion_time(self, job):
+        if (self._completion_time or
+            not slurm_state_completed(job.state)):
+            return self._completion_time
+
+        slurm_variable = [('SLURM_TIME_FORMAT', 'standard')]
+        with env.temp_environment(variables=slurm_variable):
+            completed = _run_strict(
```

Comment:
Don't run this strictly. If the command fails, simply return `None`, because otherwise logging will crash.